launch_testing_ros/examples/README.rst

```diff
@@ -0,0 +1,57 @@
+Launch testing examples
```

Comment:
meta: when https://github.com/ros-infrastructure/rosdoc2/pull/28 gets finished, it would be great to have the examples in the generated documentation.

we'll need something like the `doc` folder in launch: https://github.com/ros2/launch/tree/master/launch/doc